docs/source/metrics.rst

```diff
@@ -158,6 +158,11 @@ DiceCoefficient
 .. autoclass:: pytorch_lightning.metrics.classification.DiceCoefficient
     :noindex:
 
+EmbeddingSimilarity
+^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: pytorch_lightning.metrics.self_supervised.EmbeddingSimilarity
+
```

Comment:
```suggestion
.. autoclass:: pytorch_lightning.metrics.self_supervised.EmbeddingSimilarity
    :noindex
    
```
this may solve the docs build error